modules/psap-special-resource-operator-using-configmaps.adoc

```diff
@@ -2,10 +2,11 @@
 //
 // * hardware_enablement/psap-special-resource-operator.adoc
 
+:_content-type: PROCEDURE
 [id="deploy-simple-kmod-using-configmap-chart"]
 = Building and running the simple-kmod SpecialResource by using a config map
 
-In this example, the simple-kmod kernel module is used to show how the SRO can manage a driver container which is defined in Helm chart templates stored in a config map.
+In this example, the simple-kmod kernel module is used to show how the Special Resource Operator (SRO) can manage a driver container which is defined in Helm chart templates stored in a config map.
```

Comment:
```suggestion
In this example, the simple-kmod kernel module shows how the Special Resource Operator (SRO) manages a driver container. The driver container is defined in the Helm chart templates that are stored in a config map.
```

This sentence is going to be hard to follow for someone who is not a native speaker. Suggestion above to break it up into two sentences for simpler reading and to apply minimalism.

Reply:
Agree. I've not tried to reword/rework these sentences, just made some tweaks I needed for my changes to work.  I'll apply this and the rest of the rewording suggestions, they will make the assembly better. 

Thanks!